app/assets/stylesheets/application.css.scss

```diff
@@ -176,3 +176,14 @@ footer {
   margin: auto;
   text-align: center;
 }
+
+.posted-by {
+
+  a {
+    color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.